assets/js/script.js

```diff
@@ -0,0 +1 @@
+// TODO: remember to pseudo code!
```

Comment:
I ALREADY KNOW HOW TO PSEDUO CODE! PLEASE ADD BETTER COMMENTS.

Reply:
done